### PR TITLE
Parent key auth and key auth support added to tpm2_import

### PR DESCRIPTION
**Pull request by ichetty**

### CODE REVIEW

File: tools/tpm2_import.c [hunk 1]
```diff
@@ -146,6 +151,8 @@ static bool encrypt_seed_with_tpm2_rsa_public_key(TPM2B_DIGEST *protection_seed,
             mod_size, protection_seed->buffer, protection_seed->size, label, 10,
             tpm2_openssl_halg_from_tpmhalg(parent_name_alg), NULL);
     if (return_code != 1) {
+        LOG_ERR("parent_name_alg = 0x%X; protection_seed->size = %d; mod_size = %d",
+            parent_name_alg, protection_seed->size, mod_size);
```
[the remark this answers is not in the document]
Comment (ichetty) [reply tools/tpm2_import.c]: Will do

**Comment by williamcroberts** [comment]
@ichetty no merge commits, please use a rebase work flow and force push your topic.

File: tools/tpm2_import.c (hunk 1 above)
[the remark this answers is not in the document]
Comment (ichetty) [reply tools/tpm2_import.c]: done

**Comment by ichetty** [comment]
Rebased and force pushed and there are no conflicts and the CI build passes so closing this PR.

**Pull request closed without merging**